### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by caseyrichards** [AdguardTeam/AdGuardHome#2750] [Q&A]
Hello. Recently switched over to Adguard Home from Pi-Hole. So far so good! Two questions:
1. Is there a way to utilize the ad blocking features when I have to connect to my work’s VPN? As soon as I do that everything stops getting blocked so just want to find out how to keep it running no matter if I’m on/off the VPN in my home.
2. Is there also a way to get the same ad blocking experience on my iPhone when I’m away from home internet?
Thanks!

**Comment by ghost** [comment]
1. That defeats the purpose of a VPN, it is tough to do if at all possible because it would require split tunneling which isn't supported everywhere, and would probably anger your work IT personnel.

2.  https://github.com/rajannpatel/Pi-Hole-on-Google-Compute-Engine-Free-Tier-with-Full-Tunnel-and-Split-Tunnel-Wireguard-VPN-Configs#option-b--set-up-a-pi-hole-ad-blocking-vpn-server-behind-your-router-at-home this is for Pi-hole but the process is pretty much identical to do with AGH.

**Comment by Aikatsui** [comment]
@caseyrichards Sorry i didnt notice this issue, there's a solution but you need to wait some time. see https://github.com/AdguardTeam/AdGuardHome/issues/1769
